src/nvim/window.c

```diff
@@ -798,6 +798,16 @@ void win_config_float(win_T *wp, FloatConfig fconfig)
     fconfig.row += curwin->w_wrow;
     fconfig.col += curwin->w_wcol;
     fconfig.window = curwin->handle;
+  } else if (fconfig.relative == kFloatRelativeMouse) {
+    int row = mouse_row, col = mouse_col, grid = mouse_grid;
+    win_T *mouse_win = mouse_find_win(&grid, &row, &col);
+    if (mouse_win == NULL) {
+      return;
```

Comment:
This `return` doesn't look safe. When no window can be found I think it can be relative to editor instead?

Reply:
done